server/schemas/field.graphql

```diff
@@ -121,7 +121,9 @@ type SchemaFieldInteger {
 
 type SchemaFieldReference {
   modelId: ID!
+  correspondingSchemaId: ID
   correspondingFieldId: ID
+  correspondingField: SchemaField
```

Comment:
Also you can add some fileds (with resolver) like

```suggestion
  model: Model
  correspondingSchema: Schema
  correspondingField: SchemaField
```

Reply:
Done!